### ISSUE TITLE
Update the User Action flow chart

### ISSUE THREAD

**Issue by BrockDW** [airavata-courses/Persona#6]
1. Add a few more data flows (search, categorize, etc.)
2. Group steps in the flow into groups (for example, Login flow into the Login group, Upload flow into the Upload Group, etc.)

**Comment by BrockDW** [comment]
User Action Flow Chart is here: https://whimsical.com/flowchart-7kv1fBnQaN7G8HPeyKzjEp

Passcode is: persona

**Comment by BrockDW** [comment]
Flow chart updated

1. Added some color separation
2. Added few more functionalities

**Comment by sureshvsm123** [comment]
Yes, After segregation with colors, it is giving more value to each of our colored components

**Comment by BrockDW** [comment]
User Action Flow Chart Done
